lib/utils/group-helper.js

```diff
@@ -40,7 +40,7 @@ class GroupHelper {
         if (!arr) {
             return;
         }
-        var result = {};
+        const result = {};
```

Comment:
result is mutated

Reply:
This prevents the re-assignment and I think we will need this.

Reply:
We can add the eslint rule `prefer-const` to be 100% sure (if it's not there)

Reply:
Hmm, I'd say adding a property to a declared object is mutation but I'm probably wrong. Small thing though, nothing blocking

Reply:
Well this is JavaScript, we don't have immutable structures. In JavaScript what it counts it's just the reassignment and if the object is not reassigned, `const` is the way to go.
But you're right. But in JS world that distinction doesn't exist :D